src/js-executor/index.js

```diff
@@ -1,41 +1,23 @@
 import './index.scss'
-import Map from 'es6-map/polyfill';
```

Comment:
Do we need `es6-map/polyfill` anywhere? If not -> `$ yarn remove es6-map/polyfill`

Reply:
It's removed now.

The `yarn.lock` file changed quite a lot, since it wasn't updated last time when we changed the babel version (https://github.com/JetBrains/kotlin-playground/pull/48/commits/275d27e10ce5120528871bed00ae1ed527f6b5bf). I didn't realise that a yarn.lock is checked-in back then and used npm.

Is yarn the recommended way to manage dependencies for this project? Then we should probably update the parts of the readme that refer to `npm`. If I used yarn from the beginning, the babel version change would not have been needed at all 😊

Reply:
Yes, I always use `yarn` :)
Ah. Yes, I'll update. Thanks a lot!